File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int n) {
    string answer = "";
    int idx = 0;
    while(idx < n){
        if(idx % 2 == 0){
            answer.append("수");
        }else{
            answer.append("박");
        }
        idx++;
    }
    
    return answer;
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.java
import java.util.*;
import java.util.stream.*;

class Solution {
    public String solution(int n) {
        return IntStream.range(1, n + 1)
            .mapToObj(i -> {
                if(i % 2 == 0) {
                    return "박";
                }else{
                    return "수";
                }
            })
            .collect(Collectors.joining());
    }
}